### PR TITLE
Adjust minimum year to 1971

### PR DESCRIPTION
**Pull request by jung-hunsoo**
- If there's a case incoming date data is incorrect, adjust year to start from 1971.
- This resolves issue of #1444

### CODE REVIEW

**Comment by jung-hunsoo** [comment]
Thanks @felixdivo.
With the fixed code, the file is loaded properly. Please be understood that it's hard to share the captured image. Instead anyone can pull and test the code from https://github.com/jung-hunsoo/python-can.git@patch-1.

**Comment by felixdivo** [comment]
I don't understand why it is hard to share the captured image. Is it very large?

Further, I don't quite get why the problem occurs. `datetime.datetime(1960, 1, 1)`  works fine for me, and the change expression `systemtime[0] + (1971 - systemtime[0])` is just `1971` right? And why that boundary, it looks oddly close to [Unix time](https://en.wikipedia.org/wiki/Unix_time)?

**Comment by zariiii9003** [comment]
> I don't understand why it is hard to share the captured image. Is it very large?
> 
> Further, I don't quite get why the problem occurs. `datetime.datetime(1960, 1, 1)` works fine for me, and the change expression `systemtime[0] + (1971 - systemtime[0])` is just `1971` right? And why that boundary, it looks oddly close to [Unix time](https://en.wikipedia.org/wiki/Unix_time)?

I think this only occurs on Windows

File: can/io/blf.py [hunk 1]
```diff
@@ -119,8 +119,13 @@ def timestamp_to_systemtime(timestamp: float) -> TSystemTime:
...
 def systemtime_to_timestamp(systemtime: TSystemTime) -> float:
     try:
+        year_adjusted = (
+            systemtime[0]
+            if systemtime[0] >= 1971
+            else systemtime[0] + (1971 - systemtime[0])
+        )
         t = datetime.datetime(
-            systemtime[0],
+            year_adjusted,
```
Comment (zariiii9003) [inline can/io/blf.py:128]:
```suggestion
            max(systemtime[0], 1971),
```

This should be sufficient?

Comment (zariiii9003) [reply can/io/blf.py:128]: Also, shouldn't it be 1970? 🤔

Comment (felixdivo) [reply can/io/blf.py:128]: Yes, that were my thoughts too. According to [the docs of `datetime.datetime`](https://docs.python.org/3/library/datetime.html#datetime.datetime), small numbers (e.g., year 1960) should be possible though.

**Comment by felixdivo** [comment]
I think we should close this as it is currently stuck.

**Pull request closed without merging**